assets/src/styles/blocks/HubspotForm/HubspotFormStyles.scss

```diff
@@ -0,0 +1,179 @@
+@import "../../master-theme/assets/src/scss/base/variables";
+@import "../../master-theme/assets/src/scss/base/colors";
+@import "../../master-theme/assets/src/scss/base/mixins";
+@import "../../master-theme/assets/src/scss/base/fonts";
+
+/* Layouts */
+@import "layouts/ImageFullWidthStyles";
+
+.hubspot-form {
+  position: relative;
+  margin-bottom: 32px;
+  font-family: $roboto;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center center;
+  box-shadow: inset 0 0 0 100vw transparentize($grey-80, .45);
+
+  &.block-wide {
+    width: 100vw;
+    /* A better way is to add these changes into the WideBlocks stylesheet */
+    margin-inline-start: calc(((100vw - 100%) / 2) * -1);
+  }
+
+  & > .container {
```

Comment:
Same comment here as in the other css file, about removing the `container` class 🙂